tests/jerry/es51-profile-list

```diff
@@ -0,0 +1,301 @@
+./regression-test-issue-255.js
+./regression-test-issue-257.js
+./regression-test-issue-260.js
+./regression-test-issue-261.js
+./regression-test-issue-262.js
+./regression-test-issue-263.js
+./regression-test-issue-264.js
+./regression-test-issue-265.js
+./regression-test-issue-266.js
+./regression-test-issue-267.js
+./regression-test-issue-274.js
+./regression-test-issue-276.js
+./regression-test-issue-279.js
+./regression-test-issue-280.js
+./regression-test-issue-281.js
+./regression-test-issue-285.js
+./regression-test-issue-312.js
+./regression-test-issue-316.js
+./regression-test-issue-339.js
+./regression-test-issue-340.js
+./regression-test-issue-341.js
+./regression-test-issue-354.js
+./regression-test-issue-359.js
+./regression-test-issue-380.js
+./regression-test-issue-381.js
+./regression-test-issue-429.js
+./regression-test-issue-440.js
+./regression-test-issue-447.js
+./regression-test-issue-453.js
+./regression-test-issue-541.js
+./regression-test-issue-563.js
+./regression-test-issue-566.js
+./regression-test-issue-612.js
+./regression-test-issue-613.js
+./regression-test-issue-614.js
+./regression-test-issue-639.js
+./regression-test-issue-640.js
+./regression-test-issue-641.js
+./regression-test-issue-642.js
+./regression-test-issue-644.js
+./regression-test-issue-646.js
+./regression-test-issue-652.js
+./regression-test-issue-653.js
+./regression-test-issue-654.js
+./regression-test-issue-655.js
+./regression-test-issue-667.js
+./regression-test-issue-669.js
+./regression-test-issue-680.js
+./regression-test-issue-686.js
+./regression-test-issue-689.js
+./regression-test-issue-703.js
+./regression-test-issue-725.js
+./regression-test-issue-736.js
+./regression-test-issue-737.js
+./regression-test-issue-738.js
+./regression-test-issue-739.js
+./regression-test-issue-741.js
+./regression-test-issue-743.js
+./regression-test-issue-745.js
+./regression-test-issue-747.js
+./regression-test-issue-781.js
+./regression-test-issue-782.js
+./regression-test-issue-783.js
+./regression-test-issue-785.js
+./regression-test-issue-786.js
+./regression-test-issue-787.js
+./regression-test-issue-798.js
+./regression-test-issue-962.js
+./regression-test-issues-43-183.js
+./relational.js
+./shift.js
+./sqrt.js
+./strict.js
+./string.js
+./string-prototype-charat.js
+./string-prototype-charcodeat.js
+./string-prototype-concat.js
+./string-prototype-indexof.js
+./string-prototype.js
+./string-prototype-lastindexof.js
+./string-prototype-localecompare.js
+./string-prototype-match.js
+./string-prototype-replace.js
+./string-prototype-search.js
+./string-prototype-slice.js
+./string-prototype-split.js
+./string-prototype-substring.js
+./string-prototype-substr.js
+./string-prototype-trim.js
+./string-surrogates-concat.js
+./string-upper-lower-case-conversion.js
+./switch-case.js
+./test-new-string.js
+./this-arg.js
+./try-catch-finally.js
+./typeof.js
+./unary-plus-minus.js
+./var-decl.js
+./variables.js
+./zero-character.js
```

Comment:
Wouldn't it be much easier to have tests/jerry/es5.1 and tests/jerry/es2015 directories side-by-side, without any kind of new skip mechnism? (It would just require moving the existing tests one directory deeper. And adapting some scripts, of course.)